modules/mergesort/include/merge_sort.h

```diff
@@ -0,0 +1,18 @@
+/**
+ * @file merge_sort.h
+ * @author WARG
+ *
+ * @section LICENSE
+ *
+ *  Copyright (c) 2015, Waterloo Aerial Robotics Group (WARG)
+ *  All rights reserved.
+ *
+ *  inthis software is licensed under a modified version of the BSD 3 clause license
+ *  that should have been included with this software in a file called COPYING.txt
+ *  Otherwise it is available at:
+ *  https://raw.githubusercontent.com/UWARG/computer-vision/master/COPYING.txt
+ */
+ #include <vector>
+ // @brief uses mergesort to sort the vector that is inputted
```

Comment:
Doxygen requires that you use doc-style comments, either `/** */` such as in the file header or `///` if you want to use c++ style comments. If you don't it won't use the comment in the generated documentation.